### PR TITLE
Merge apis endpoints

### PR DESCRIPTION
**Pull request by flicker-harmony**
## Issue

There is a need to merge explorer and sdk api endpoints and also make api available for only explorer node (for now I didn't change availability, which can be done trivially).

## Test

Localnet testing, but I didn't test very deeply because of high priority, so be careful.

## TODO

Make SDK available only for explorer node, but get feedback from other engineers and testing on betanet.

### CODE REVIEW

File: api/service/explorer/service.go [hunk 1]
```diff
@@ -323,6 +323,112 @@ func (s *Service) GetExplorerBlocks(w http.ResponseWriter, r *http.Request) {
...
+					utils.Logger().Warn().Err(err).Msg("cannot read committee for new epoch")
+				}
+			} else {
+				state, err := accountBlock.Header().GetShardState()
+				if err == nil {
+					for _, shardCommittee := range state {
+						if shardCommittee.ShardID == accountBlock.ShardID() {
+							committee = &shardCommittee
+							break
+						}
```
[the remark this answers is not in the document]
Comment (flicker-harmony) [reply api/service/explorer/service.go]: I can wrap it into a method, but I can't just change the logic.

**Comment by LeoHChen** [comment]
Is it working in your test?

**Comment by flicker-harmony** [comment]
testing okay, explorer_getExplorerBlocks for example
Screenshot 2019-09-15 at 02 00 04

**Comment by flicker-harmony** [comment]
Jenkins, test this please.

**Comment by LeoHChen** [comment]
Jenkins, test this please.

**Comment by LeoHChen** [comment]
You need to rebase your code and please squash you change into just one commit. Other than those, it looks good to me.

**Comment by flicker-harmony** [comment]
Jenkins, test this please.

**Comment by LeoHChen** [comment]
Jenkins, test this please.

**Pull request merged**